docs/configuration.md

```diff
@@ -703,71 +613,25 @@ it supports any interesting structures desired, leaving it up to the middleware
 initialization function to best determine how to handle the specific
 interpretation of the options.
 
-### cloudfront
-
-<table>
-  <tr>
-    <th>Parameter</th>
-    <th>Required</th>
-    <th>Description</th>
-  </tr>
-  <tr>
-    <td>
-      <code>baseurl</code>
-    </td>
-    <td>
-      yes
-    </td>
-    <td>
-      <code>SCHEME://HOST[/PATH]</code> at which Cloudfront is served.
-    </td>
-  </tr>
-    <tr>
-    <td>
-      <code>privatekey</code>
-    </td>
-    <td>
-      yes
-    </td>
-    <td>
-      Private Key for Cloudfront provided by AWS.
-    </td>
-  </tr>
-    <tr>
-    <td>
-      <code>keypairid</code>
-    </td>
-    <td>
-      yes
-    </td>
-    <td>
-      Key pair ID provided by AWS.
-    </td>
-  </tr>
-    <tr>
-    <td>
-      <code>duration</code>
-    </td>
-    <td>
-      no
-    </td>
-    <td>
-      Specify a `duration` by providing an integer and a unit. Valid time units are `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`. For example, `3000s` is a valid duration; there should be no space between the integer and unit. If you do not specify a `duration` or specify an integer without a time unit, this defaults to 20 minutes.
-    </td>
-  </tr>
-</table>
-
-### redirect
-
-In place of the `cloudfront` storage middleware, the `redirect`
-storage middleware can be used to specify a custom URL to a location
-of a proxy for the layer stored by the S3 storage driver.
-
-| Parameter | Required | Description |
-| --- | --- | --- |
-| baseurl   | yes      | `SCHEME://HOST` at which layers are served. Can also contain port. For example, `https://example.com:5443`. |
-
-## reporting
+### `cloudfront`
+
+
+| Parameter | Required | Description                                           |
+|-----------|----------|-------------------------------------------------------|
+| `baseurl` | yes      | The `SCHEME://HOST[/PATH]` at which Cloudfront is served. |
+| `privatekey` | yes   | The private key for Cloudfront, provided by AWS.        |
+| `keypairid` | yes    | The key pair ID provided by AWS.                         |
+| `duration` | no      | An integer and unit for the duration of the Cloudfront session. Valid time units are `ns`, `us` (or `µs`), `ms`, `s`, `m`, or `h`. For example, `3000s` is valid, but `3000 s` is not. If you do not specify a `duration` or you specify an integer without a time unit, the duration defaults to `20m` (20 minutes).|
+
+### `redirect`
+
+You can use the `redirect` storage middleware to specify a custom URL to a
+location of a proxy for the layer stored by the S3 storage driver.
+
+|-----------|----------|-------------------------------------------------------------------------------------------------------------|
+| `baseurl` | yes      | `SCHEME://HOST` at which layers are served. Can also contain port. For example, `https://example.com:5443`. |
+
+## `reporting`
 
```

Comment:
This must have been part of a different change in master, but in master this is surrounded by ` ``` `